launcher/src/main.rs

```diff
@@ -508,7 +508,7 @@ struct Args {
     /// Whether to use the prefix caching mechanism. This will skip computing attention on previously cached prefixes
     /// in the prompt. Useful in cases where many queries need to be run over a shared context, or for long multi-turn
     /// chats conversations.
-    #[clap(long, env)]
+    #[clap(default_value = "true", long, env)]
```

Comment:
Are we confident this is stable enough to enable by default? I believe there are some open questions on this at the moment. Maybe we can start with just landing the eager prefill change.